tests/sweep_framework/sweeps/matmul/short/matmul_default_sharded.py

```diff
@@ -28,43 +28,43 @@ class TensorMemoryConfigs(enum.Enum):
         "batch_sizes": [(2, 3)],
         "input_shapes": [(1600, 224, 896)],
         "batch_matrix_multiply": [False],
-        "input_a_sharded_memory_config_specs": [TensorMemoryConfigs.BLOCK_Y5_X7],
+        "input_a_sharded_memory_config_specs": [TensorMemoryConfigs.BLOCK_Y5_X7.name],
```

Comment:
Is using .name more stable than just comparing the enums directly? Or is there some other reason to use it instead of comparing the enum directly?

Reply:
input_a_sharded_memory_config_specs/input_b_sharded_memory_config_specs which are passed in as config in get_config_dict() are strings, not enums.

Therefore e.g. input_a_sharded_memory_config_specs is "TensorMemoryConfigs.BLOCK_Y5_X7" instead of TensorMemoryConfigs.BLOCK_Y5_X7

The name of the enum is just "BLOCK_Y5_X7"

Therefore neither `elif config == TensorMemoryConfigs.BLOCK_Y5_X7:` nor `TensorMemoryConfigs.BLOCK_Y5_X7.name` would work.

Instead I'd have to use either name or value. I figured name would be easier to see in the DBs.

I'm guessing that for some reason the enums are not converted properly via json, but I'm not sure what would be happening there.

Reply:
I see, it would be because that TensorMemoryConfigs is locally defined here, so the deserialization has no way to know of that type. This works then.